### PR TITLE
Visibilities grouped by brokers

### PR DESCRIPTION
**Pull request by pankrator**

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -35,7 +35,7 @@
 
 [[constraint]]
   name = "github.com/Peripli/service-broker-proxy"
-  version = "=0.2.0"
+  branch = "visibilities-brokers"
```
Comment (NickyMateev) [inline Gopkg.toml]: Should be reverted when the **service-broker-proxy** PR is merged

File: application.yml [hunk 2]
```diff
@@ -1,8 +1,10 @@
 # Any of the below config properties can be provided via Environment Variables
 app:
-  url: http://10.0.2.2
+  url: http://10.0.2.2:8081
```
Comment (dpanayotov) [inline application.yml]: revert file

File: cf/service_access.go [hunk 3]
```diff
@@ -140,21 +139,32 @@ func (pc *PlatformClient) UpdateServicePlan(planGUID string, request ServicePlan
...
+	brokers, err := pc.getBrokersByName([]string{brokerName})
 	if err != nil {
 		return cfclient.ServicePlan{}, wrapCFError(err)
 	}
-	if len(plans) == 0 {
-		return cfclient.ServicePlan{}, errors.Errorf("zero plans with catalog plan GUID = %s found", catalogPlanGUID)
+	if len(brokers) == 0 {
+		return cfclient.ServicePlan{}, errors.Errorf("no brokers found for broker name %s", brokerName)
+	}
+	if len(brokers) > 1 {
```
Comment (dpanayotov) [inline cf/service_access.go]: not sure if this should be the case. Yes, the sbproxies' broker names are unique but that might not be the case in the future. It's ok for now

File: cf/service_visibilities.go [hunk 4]
```diff
@@ -158,28 +273,77 @@ func (pc *PlatformClient) getPlansVisibilities(ctx context.Context, plans []cfcl
...
+}
+
+func (q *queryBuilder) build() map[string][]string {
+	queryComponents := make([]string, 0)
+	for key, params := range q.filters {
+		component := fmt.Sprintf("%s IN %s", key, params)
+		queryComponents = append(queryComponents, component)
+	}
+	query := strings.Join(queryComponents, ";")
+	log.D().Debugf("CF filter query built: %s", query)
```
Comment (dpanayotov) [inline cf/service_visibilities.go]: this could use the context logging

File: cf/service_access.go (hunk 3 above)
Comment (pankrator) [reply cf/service_access.go]: Should we add some comment here, that we are aware it might break in the future?

File: Gopkg.toml (hunk 1 above)
Comment (dpanayotov) [reply Gopkg.toml]: reference release

File: cf/service_access.go (hunk 3 above)
Comment (dpanayotov) [reply cf/service_access.go]: an issue might be better

Comment (pankrator) [reply cf/service_access.go]: created

**Pull request merged**